client/js/loading-slow-alert.js

```diff
@@ -8,40 +8,53 @@
  *    so that the timeout can be triggered while slow JS is loading
  */
 
-setTimeout(function() {
-	var element = document.getElementById("loading-slow");
+function displayReload() {
+	var loadingReload = document.getElementById("loading-reload");
+	if (loadingReload) {
+		loadingReload.style.display = "block";
+	}
+}
+
+var loadingSlowTimeout = setTimeout(function() {
```

Comment:
This creates a global variables (along with the function above). Wrap this entire script in an anonymous function.

Reply:
Done.